x-pack/plugin/core/src/main/resources/ilm-history.json

```diff
@@ -80,6 +80,7 @@
         }
       }
   },
+  "allow_auto_create": true,
```

Comment:
I think that `allow_auto_create` doesn't need to be defined? Since auto creation is allowed by default?

Reply:
This is to handle the situation where `auto_create_index` is set to false (or some restrictive pattern). In that case, having `allow_auto_create` set to true means that ES will still create the index, overriding `auto_create_index`. Is the name of this new field confusing, maybe?

Reply:
No, the name is not confusing. I forgot, that the new setting can also be used to overwrite anything that is specified in `auto_create_index` setting.